docs-chef-io/content/server/ctl_chef_server.md

```diff
@@ -659,13 +659,14 @@ chef-server-ctl show-service-credentials
 
 ### cleanup-bifrost
 
-The `cleanup-bifrost` subcommand removes unused authorization objects
-from the authorization database (called bifrost). These unused objects
-can accumulate on long-running Chef Infra Servers as a result of failed object
-creation requests. For most users, the unused authorization objects do
+The `cleanup-bifrost` subcommand removes unused authorization ids of the actors and the objects
+from the authorization database (called bifrost). These unused authorization ids of actors
+can accumulate on long-running Chef Infra Servers as a result of failed actor
```

Comment:
can they also accumulate due to deletes?

Reply:
no, its only during the cookbook upload phase. irrespective of whether we delete the cookbook or not the orphan authz id accumulated while cookbook upload phase wont be deleted.